src/CallingExtensions.js

```diff
@@ -102,9 +102,10 @@ class CallingExtensions {
     });
   }
 
-  callAnswered() {
+  callAnswered(data) {
```

Comment:
Could you please add a JSdoc description for `data`, similar to the [publishToChannel](https://github.com/HubSpot/calling-extensions-sdk/pull/212/files#diff-1704fbaf2f006ebd272eb5b33d75368885491952cc7b22b07b30666b71f3cab6R196) event.

Reply:
done